src/app/query-builder/page.tsx

```diff
@@ -17,50 +39,103 @@ export interface SearchParamsProps {
 }
 
 export default function QueryBuilder() {
+  const searchParams = useSearchParams();
+  const measureBundle = useRecoilValue(measureBundleState);
+  const bulkExportBaseURL = useRecoilValue(bulkServerURLState);
+  const [groupId, setGroupId] = useState(null);
+
+  const createGroup = () => {
+    fetch(`${bulkExportBaseURL}/Group`, {
+      method: 'POST',
+      body: measureBundle.groupText,
+      headers: { 'Content-Type': 'application/json+fhir' }
+    })
+      .then(response => response.json())
+      .then(resObj => {
+        setGroupId(resObj.id);
+        showNotification({
+          icon: <IconCircleCheck />,
+          title: 'Group Creation Success',
+          message: `Successfully created group at id ${resObj.id}`,
+          color: 'green'
+        });
+      })
+      .catch(err => {
+        console.error(err);
+        showNotification({
+          icon: <IconAlertCircle />,
+          title: 'Group creation failed',
```

Comment:
```suggestion
          title: 'Group Creation Failed',
```
Doesn't matter, but for consistency ?